pom.xml

```diff
@@ -70,7 +70,7 @@
     <analyzerCommons.version>1.22.0.848</analyzerCommons.version>
 
     <artifactsToPublish>${project.groupId}:sonar-html-plugin:jar</artifactsToPublish>
-    <maven.compiler.release>8</maven.compiler.release>
+    <jdk.min.version>11</jdk.min.version>
```

Comment:
Looks good to me.
Though I can't find documentation for the property you used (`jdk.min.version`). Maybe I'll need a POM/Maven refresher :)